src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/DisputeController.java

```diff
@@ -393,29 +393,29 @@ public ResponseEntity<Long> saveDisputantUpdateRequest(
 	}
 
 	/**
-	 * Get endpoint that retrieves all <code>DisputantUpdateRequest</code>s from persistent storage that is associated with the given Dispute via disputeId.
+	 * Get endpoint that retrieves all <code>DisputantUpdateRequest</code>s from persistent storage that is associated with the optionally given Dispute via optional disputeId.
 	 *
 	 * @param disputantUpdateRequest id to be retrieved
 	 * @return id of the saved {@link DisputantUpdateRequest}
 	 */
-	@GetMapping("/dispute/{id}/updateRequest")
-	@Operation(summary = "An endpoint that retrieves all DisputantUpdateRequest for a given Dispute, optionally filtered by status.")
+	@GetMapping("/dispute/updateRequests")
```

Comment:
I would be nice to a Junit test added to the test suite for this endpoint. Something in the DisputeControllerTest that uses the mvc.perform().

Reply:
added

Reply:
added